tests/components/whirlpool/test_sensor.py

```diff
@@ -0,0 +1,180 @@
+"""Test the Whirlpool Sensor domain."""
+from unittest.mock import MagicMock
+
+from attr import dataclass
+from whirlpool.washerdryer import MachineState
+
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers import entity_registry
+
+from . import init_integration
+
+
+async def update_sensor_state(
+    hass: HomeAssistant,
+    entity_id: str,
+    mock_sensor_api_instance: MagicMock,
+):
+    """Simulate an update trigger from the API."""
+
+    for call in mock_sensor_api_instance.register_attr_callback.call_args_list:
+        update_ha_state_cb = call[0][0]
+        update_ha_state_cb()
+        await hass.async_block_till_done()
+
+    return hass.states.get(entity_id)
+
+
+def side_effect_function_open_door(*args, **kwargs):
+    """Return correct value for attribute."""
+    if args[0] == "Cavity_TimeStatusEstTimeRemaining":
+        return 3540
+
+    if args[0] == "Cavity_OpStatusDoorOpen":
+        return "1"
+
+    if args[0] == "WashCavity_OpStatusBulkDispense1Level":
+        return "3"
+
+
+async def test_sensor_values(
+    hass: HomeAssistant,
+    mock_sensor_api_instances: MagicMock,
+    mock_sensor1_api: MagicMock,
+    mock_sensor2_api: MagicMock,
+    # mock_sensor_api: MagicMock,
```

Comment:
Don't check in commented out code

Reply:
Those darn "quick" comments have a way for hiding :)